container_pipeline/workers/scan.py

```diff
@@ -70,6 +70,8 @@ def handle_job(self, job):
         if self.job.get("weekly"):
             scanners_data["action"] = "notify_user"
             self.queue.put(json.dumps(scanners_data), 'master_tube')
+            self.init_next_phase_data('delivery')
+            self.logger.debug("Put job for delivery on master tube")
```

Comment:
Even though we have limitation from db fields point of view, surely we dont have one for the log messages. :smile: 
```
self.logger.debug("Weekly scan for {project }is complete.")
```
should fit better here.